x/datapool/keeper/pool.go

```diff
@@ -615,15 +617,86 @@ func (k Keeper) GetDataPassRedeemReceipt(ctx sdk.Context, poolID, round, nftID u
 	return receipt, nil
 }
 
-func (k Keeper) SetDataPassRedeemReceipt(ctx sdk.Context, redeemReceipt types.DataPassRedeemReceipt) error {
+func (k Keeper) SetDataPassRedeemReceipt(ctx sdk.Context, redeemReceipt types.DataPassRedeemReceipt) {
```

Comment:
This keeper function doesn't need to return error, so I fixed. Please check this as well, @inchori

Reply:
Thanks for fixing these codes! 🤩